sklearn/preprocessing/data.py

```diff
@@ -1917,3 +1920,38 @@ def transform(self, X):
         """
         return _transform_selected(X, self._transform,
                                    self.categorical_features, copy=True)
+
+
+def _boxcox(X):
```

Comment:
Could you please provide how to achieve that since I couldn't find a way to
extract the first part of the tuple output?
On 15 May 2016 7:29 pm, "Raghav R V" <notifications@github.com> wrote:

> In sklearn/preprocessing/data.py
> <https://github.com/scikit-learn/scikit-learn/pull/6781#discussion_r63292049>
> :
>
> > @@ -1917,3 +1920,38 @@ def transform(self, X):
> >          """
> >          return _transform_selected(X, self._transform,
> >                                     self.categorical_features, copy=True)
> > +
> > +
> > +def _boxcox(X):
>
> But I think you could pass the stats.boxcox as such and extract the
> desired output in one go? Would be cleaner? No strong opinion though.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/scikit-learn/scikit-learn/pull/6781/files/14f14c70b82d373a4a6b7f28356bebd1d17b5f56#r63292049>
>